### ISSUE TITLE
Add the information about advanced search features to webpage

### ISSUE THREAD

**Issue by ShalokShalom** [mediathekview/mediathekviewweb#61]
I suggest to add this [information about the search feature](https://github.com/mediathekview/mediathekviewweb#erweiterte-suche) to the webpage.

**Comment by bagbag** [comment]
As in the other Issues: "will be done with the new client", as current code is "not good", which leads to implementing any new thing being stupid.

**Comment by ShalokShalom** [comment]
ok, thanks :)

**Comment by bagbag** [comment]
You can leave it open to have a reminder.

**Comment by stefan123t** [comment]
This is solved in the current search dialog, the on mouse over tooltip has the extended search features:

<img width="502" height="325" alt="Image" src="https://github.com/user-attachments/assets/3edbab85-d544-492d-80c4-38c85b39b04c" />

**Comment by stefan123t** [comment]
@ShalokShalom image has been attached in github, as you probably noticed already.
Many thanks for your valuable work on MediathekViewWeb !

**Comment by ShalokShalom** [comment]
Yeah, I think there was an issue with GitHub's server, it didn't load the image. Thanks
